specification/schemaregistry/SchemaRegistry/main.tsp

```diff
@@ -0,0 +1,282 @@
+  is OAuth2Auth<[
+    {
+      type: OAuth2FlowType.implicit;
+      authorizationUrl: "https://login.microsoftonline.com/common/oauth2/authorize";
+
+      // add description to below: "impersonate your user account"
+      scopes: ["https://eventhubs.azure.net/.default"];
+    }
+  ]>;
+
+// ROUTES
+alias resourceOperations = Azure.Core.StandardResourceOperations;
+
+interface SchemaOperations {
+  //  GET https://{namespaceName}.servicebus.windows.net/$schemaGroups?api-version={apiVersion}
+  @summary("Get list of schema groups.")
+  @doc("Gets the list of schema groups user is authorized to access.")
+  @sharedRoute
+  listSchemaGroups is resourceOperations.ResourceList<SchemaGroup>;
+
+  //  GET https://{namespaceName}.servicebus.windows.net/$schemaGroups/$schemas/{id}?api-version={apiVersion}
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Standard read operation w/ SchemaId resource doesn't allow returning binary schema and headers."
+  @summary("Get a registered schema by its unique ID reference.")
+  @doc("Gets a registered schema by its unique ID.  Azure Schema Registry guarantees that ID is unique within a namespace. Operation response type is based on serialization of schema requested.")
+  @route("/$schemaGroups/$schemas/{id}")
+  @get
+  getSchemaById is Azure.Core.Foundations.Operation<
+    Azure.Core.Foundations.ItemKeysOf<SchemaId>,
+    OkResponse & SchemaHeaders & SchemaContent & SchemaContentType
+  >;
+
+  //  GET https://{namespaceName}.servicebus.windows.net/$schemaGroups/{groupName}/schemas/{name}/versions/?api-version={apiVersion}
+  @summary("List schema versions.")
+  @doc("Gets the list of all versions of one schema.")
+  listSchemaVersions is resourceOperations.ResourceList<Version>;
+
+  //  GET https://{namespaceName}.servicebus.windows.net/$schemaGroups/{groupName}/schemas/{name}/versions/{schemaVersion}?api-version={apiVersion}
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Standard read operation w/ Version resource doesn't allow returning non-version response."
+  @summary("Get specific schema versions.")
+  @doc("Gets one specific version of one schema.")
+  @route("/$schemaGroups/{groupName}/schemas/{name}/versions/{schemaVersion}")
+  @get
+  getSchemaByVersion is Azure.Core.Foundations.Operation<
+    Azure.Core.Foundations.ItemKeysOf<Version>,
+    OkResponse & SchemaHeaders & SchemaContent & SchemaContentType
+  >;
+
+  //  POST https://{namespaceName}.servicebus.windows.net/$schemaGroups/{groupName}/schemas/{name}:get-id?api-version={apiVersion}
+  @summary("Get ID for existing schema.")
+  @doc("Gets the ID referencing an existing schema within the specified schema group, as matched by schema content comparison.")
+  @action("get-id")
+  getSchemaIdByContent is resourceOperations.ResourceAction<
+    SchemaNameContent,
+    {
+      ...SchemaContentType;
+
+      @doc("String representation (UTF-8) of the registered schema.")
+      @body
```

Comment:
test removing this + generating

Reply:
nothing relevant changes. The following JSON type alias is added to _models, but isn't used anywhere:
`JSON = MutableMapping[str, Any]`